packages/devtools/lib/src/ui/ui_utils.dart

```diff
@@ -254,16 +254,25 @@ class RegisteredServiceExtensionButton {
   }
 }
 
-bool tabDisabledByQuery(String key) {
+Set _hiddenPages;
+
+Set get hiddenPages {
+  return _hiddenPages ??= _lookupHiddenPages();
+}
+
+Set _lookupHiddenPages() {
   final queryString = html.window.location.search;
   if (queryString == null || queryString.length <= 1) {
-    return false;
+    return null;
```

Comment:
return an empty set instead. The code as written will throw exceptions.

Reply:
done.